R/learner_lme4_regr_lmer.R

```diff
@@ -15,7 +15,34 @@
 #' `r format_bib("bates2010lme4")`
 #'
 #' @template seealso_learner
-#' @template example
+#' @examples
+#' if (requireNamespace(c("lme4", "faraway"), quietly = TRUE)) {
```

Comment:
why is faraway neeed here?

Reply:
for building the task specified for this learner:
# Define a Task
#' task = mlr3::as_task_regr(lme4::sleepstudy, target = "Reaction")